### PR TITLE
Add win_set_powercfg.yml in Windows test setup

### PR DESCRIPTION
**Pull request by Tomorrow9**
Signed-off-by: dianew <dianew@vmware.com>

### CODE REVIEW

File: windows/setup/test_setup.yml [hunk 1]
```diff
@@ -91,28 +91,37 @@
...
-- debug:
-    msg: "VM files path: {{ vm_files_path }}"
+# Set VM files path on ESXi host
+- block:
+    - include_tasks: ../../common/vm_get_guest_facts.yml
+    - name: Set fact of the VM files path
+      set_fact:
+        vm_files_path: "/vmfs/volumes/{{ vm_guest_facts.instance.hw_files[0].split(' ')[0].strip('[]') }}/{{ ('/').join(vm_guest_facts.instance.hw_files[0].split(' ')[1].split('/')[:-1]) }}"
+    - debug:
+        msg: "VM files path: {{ vm_files_path }}"
```
Comment (keirazhang) [inline windows/setup/test_setup.yml:126]: I am afraid this debug task won't be executed after vm_files_path is set, as it doesn't meet the condition any more. I think we can move it out of the block.

Comment (Tomorrow9) [reply windows/setup/test_setup.yml]: Yeah, right, that's the problem. Thanks.

**Review by keirazhang — APPROVED** [review]
LGTM. Thanks!

**Comment by Tomorrow9** [comment]
Fixes: #33

**Pull request merged**